bucket/zigup.json

```
@@ -0,0 +1,16 @@
+{
+    "version": "2024_02_25",
+    "description": "Download and manage Zig compilers",
+    "homepage": "https://github.com/marler8997/zigup",
+    "license": {
+        "identifier": "MIT-0",
+        "url": "https://github.com/marler8997/zigup/blob/master/LICENSE"
+    },
+    "url": "https://github.com/marler8997/zigup/releases/download/v2024_02_25/zigup.windows-latest-x86_64.zip",
+    "hash": "9aa5c98b55203712af61242753871ad75cada479373713b4dc2804c21c4db561",
+    "bin": "zigup.exe",
+    "checkver": "github",
```

```suggestion
    "checkver": {
        "url": "https://api.github.com/repos/marler8997/zigup/releases",
        "regex": "v([\\d_]+)"
    },
```
This should fix the checkver, it wasn't working because of underscores in the version number